### PR TITLE
Math magicians: full website

### PR DESCRIPTION
**Pull request by quavo19**
### In this milestone;
- I made a navigation bar for my page
- I also added more pages to my page, such as a Quotes page and a Home page.
- I made sure that for every page there is a route

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,23 @@
 import './Style.css';
+import { Routes, Route, BrowserRouter } from 'react-router-dom';
 import Calculator from './Components/Calculator';
+import Navbar from './Components/Nav';
+import HomePage from './Components/Home';
+import Quotes from './Components/Quotes';
```
Comment (DammyShittu) [inline src/App.js]: - [ ] I could not run your app properly due to this error in the image.

  image

  You need to be mindful of your file name and its casing. You have `components` with small letter, but you are trying to import using capitalized name `Components`. Kindly fix this.

  ```
  import Calculator from './components/Calculator';
  import Navbar from './components/Nav';
  import HomePage from './components/Home';
  import Quotes from './components/Quotes';
  ```

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @quavo19,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (quavo19) [reply src/App.js]: working on that sir

**Review by Reem-lab — APPROVED** [review]
# Hi @quavo19, 👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
### Your design is very cool, I like it great job 🔥🔥


## Highlights
- No Linters error ✅
- Followed Gitflow ✅
- Great working on the design ✅
- Professional README ✅🔥
- Descriptive PR ✅
- Great working on the Routes functionality ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**